src/RageSoundPosMap.cpp

```diff
@@ -180,17 +180,17 @@ int64_t pos_map_queue::Search( int64_t iSourceFrame, bool *bApproximate ) const
 	 * 3. Underflow; we'll be given a larger frame number than we know about.
 	 */
 #if defined(WIN32)
-#define LI "%I64i"
-#elif defined(PRIi64)
-#define LI "%" PRIi64
+#define I64F "%I64i"
+#elif defined(__x86_64__)
+#define I64F "%li"
```

Comment:
I want to make sure I understand this part of the code right. We are logging numbers that are meant to be cast as 64-bit numbers? Why would that matter when our ints are (most likely) 32-bit?

I wonder if the `cppformat` library in `5_1_0` would make this easier to address.